### ISSUE TITLE
Minor bug in the website example of the Hubbard model DMFT script

### ISSUE THREAD

**Issue by donrolih** [TRIQS/nrgljubljana_interface#3]
### Description

Running the example script for the Hubbard model DMFT given on the [webpage](https://triqs.github.io/nrgljubljana_interface/latest/guide/hubbard.html) throws an error. 

### Steps to Reproduce

1. Run ``` python hubbard.py ```

**Expected behavior:** Start the DMFT loop.

**Actual behavior:** Running the script throws the following error

```
Traceback (most recent call last):
  File "hubbard.py", line 75, in <module>
    for k,v in mp.items():
RuntimeError: dictionary changed size during iteration
```
Line 75 is the following piece of code
```
# Model Parameters
mp = { "U1": U, "B1": B, "omega": omega, "g1": g1, "n1": n1 }
for k,v in mp.items():
  if v is None: del mp[k] # Remove undefined parameters from the dictionary
sp["model_parameters"] = mp
```
The problem is in changing the dictionary while iterating over it.

### Versions
NRGLjubljana_interface version:

``` You are using nrgljubljana_interface version 3.0.0 ```

### Solution
The bug is easily fixed: the dictionary should not be changed while it is being iterated over. We can iterate over a copy:
```
# Model Parameters
mp = { "U1": U, "B1": B, "omega": omega, "g1": g1, "n1": n1 }
for k,v in mp.copy().items():
  if v is None: del mp[k] # Remove undefined parameters from the dictionary
sp["model_parameters"] = mp
```

**Comment by Wentzell** [comment]
Thank you @donrolih for pointing this out. Could you create a PR including this fix?

**Comment by donrolih** [comment]
@Wentzell I have created a pull request #4 for this.

**Comment by Wentzell** [comment]
This is fixed after the merge of #4
